packages/bento-design-system/src/Table/types.ts

```diff
@@ -25,7 +26,7 @@ declare module "react-table" {
 
 export type Column<A extends string, D extends object, V> = {
   accessor: A;
-  Cell: (props: CellProps<D, V>) => Children;
+  Cell: (props: CellProps<D, V>) => ReturnType<FunctionComponent>;
```

Comment:
This is due to a slight change in the definition of `FunctionComponent`, which is ultimately used by the `Renderer` type in react-table.

It's technically breaking at the type-level, since we don't allow `undefined` to be returned anymore by the `Cell` function, but I think we can ignore this in practice.